### ISSUE TITLE
Character articles go from one world to another

### ISSUE THREAD

**Issue by KillerKonijn** [worldanvil/worldanvil-bug-tracker#940]
**World Anvil Username**:DM-KillerRabbit

**World / Campaign / RPG Character / Manuscript Title**: Peneopia ; Icewind Dale ; ...

**Describe the Issue**
I host multiple D&D campaigns, and I use World Anvil for all my campaigns. I make different worlds for each game.
Some users play in multiple worlds, so they have 1 character for world x and 1 character for world y. When they go from one world to the other, their character article get transported from one world to the other, so the article is not available in the world it should be.

For example; Jondar is a character which plays in Icewind Dale. When he is looking at Icewind Dale, and opens another tabblad to check the campaign world Peneopia, instead of looking with his character Suzie, now he is with Jondar in this world. 

This is clearly not as it is intended, and is quite annoying. Is there any fix to it?

**Comment by iamromeo** [comment]
This is not a bug, this is how World Anvil works. The articles are meant to move from one world to another if the character switches between worlds. 

This is not a bug, but if you have a suggested solution do tell.

**Comment by KillerKonijn** [comment]
Hello

But the character isn't switching, it's the account that has a character in world 1 and another in world 2...

Van: Dimitris Romeo Havlidis ***@***.***>
Verzonden: donderdag 24 maart 2022 11:50
Aan: worldanvil/worldanvil-bug-tracker ***@***.***>
CC: KillerKonijn ***@***.***>; Author ***@***.***>
Onderwerp: Re: [worldanvil/worldanvil-bug-tracker] Character articles go from one world to another (Issue #940)


This is not a bug, this is how World Anvil works. The articles are meant to move from one world to another if the character switches between worlds.

This is not a bug, but if you have a suggested solution do tell.

-

**Comment by Kordishal** [comment]
Yeah I think you misunderstood the issue here Dimitris. But this is probably related to the active character. Does this happen if the user switches the active character before looking at the other world?

**Comment by iamromeo** [comment]
I am really sorry but I still do not understand what you mean

**Comment by KillerKonijn** [comment]
Hello

Yes, that's indeed what happens. The player involved (but its with other players aswell), answered me:
I changed it. It should be okay now. What I did. 1. logging in World Anvil and view the character I want to place in Icewind Dale (world 1) (Jondar = character 1)). Go to the link provided for access to Icewind Dale (world 1). And voila Jondar is back  in Icewind.
The same process works for Suzie (character 2 for world 2).

So if for example, player is looking to world 1 with character one, and than opens another link that leads to world 2, character 1 will be moved.
Also, if you open a second tab to look at world 2

  *   While already in another tab with player 1 on world 1
  *   On second tab, you will now be in world 2 with character 1
  *   If you go back to first tab, and go to another article on world 1, character 1 will be suddenly in world 2, and you will be looking at world 2, and can't go to other articles from world 1

Its hard to explain clearly.
For now all my players need to be very mindful not to browse between 2 worlds, but always close everything before switching, to avoid this mix up.

Thanks for looking into it

Van: Jonas Waeber ***@***.***>
Verzonden: vrijdag 25 maart 2022 16:18
Aan: worldanvil/worldanvil-bug-tracker ***@***.***>
CC: KillerKonijn ***@***.***>; Author ***@***.***>
Onderwerp: Re: [worldanvil/worldanvil-bug-tracker] Character articles go from one world to another (Issue #940)


There are two campaigns and two characters.

Character 1 is in Campaign 1
Character 2 is in Campaign 2

Both Characters are owned by the same account owned by User X.

When now User X is looking at articles from Campaign 1 with Character 1 active that works fine.

But when now User X opens an article from Campaign / World 2 while still on Character 1 the article of that character is moved to Campaign 2, but Character 1 is still supposed to be in Campaign 1.

I have not had time yet to test this.

-
